app.js

```diff
@@ -1,14 +1,43 @@
-import express from 'express';
+const express = require("express");
 const app = express();
+const regeneratorRuntime = require("regenerator-runtime");
```

Comment:
I'm confused as to why we had to do this even with babel installed and (as far as we can tell) configured correctly. Let's remember to ask instructors about this.

Reply:
Yes, let's try to get back to import/export syntax if we can!